apps/meteor/app/apps/server/communication/methods.ts

```diff
@@ -1,49 +1,54 @@
 import { Meteor } from 'meteor/meteor';
+import { SettingValue } from '@rocket.chat/core-typings';
 
 import { Settings } from '../../../models/server/raw';
 import { hasPermission } from '../../../authorization/server';
 import { twoFactorRequired } from '../../../2fa/server/twoFactorRequired';
+import { AppServerOrchestrator } from '../orchestrator';
 
-const waitToLoad = function (orch) {
-	return new Promise((resolve) => {
+const waitToLoad = function (orch: AppServerOrchestrator): unknown {
+	return new Promise<void>((resolve) => {
 		let id = setInterval(() => {
 			if (orch.isEnabled() && orch.isLoaded()) {
 				clearInterval(id);
-				id = -1;
+				id = -1 as unknown as NodeJS.Timeout;
 				resolve();
 			}
 		}, 100);
 	});
 };
 
-const waitToUnload = function (orch) {
-	return new Promise((resolve) => {
+const waitToUnload = function (orch: AppServerOrchestrator): unknown {
+	return new Promise<void>((resolve) => {
 		let id = setInterval(() => {
 			if (!orch.isEnabled() && !orch.isLoaded()) {
 				clearInterval(id);
-				id = -1;
+				id = -1 as unknown as NodeJS.Timeout;
 				resolve();
 			}
 		}, 100);
 	});
 };
 
 export class AppMethods {
-	constructor(orch) {
+	_orch: AppServerOrchestrator;
```

Comment:
It will be nicer if all class properties have their proper mark either: 'private', 'public' or 'protected'